packages/components/src/InputFile/InputFile.css

```diff
@@ -22,3 +22,13 @@
   border-color: var(--color-focus);
   background-color: var(--color-surface--hover);
 }
+
+.dropZone.error {
+  border: var(--color-critical) dashed var(--border-thick);
+}
+
+.validationErrors {
+  display: flex;
+  justify-content: center;
```

Comment:
What is the `justify-content` doing here?

Reply:
That doesn't need to be there anymore. It was used to get the validations to be in the center of the component.